.ci/scripts/go-plus/github-differ/test_tools.sh

```diff
@@ -2,10 +2,11 @@
 
 MM_LOCAL_PATH=$1
 TPG_LOCAL_PATH=$2
```

Comment:
Should we remove `TPG_LOCAL_PATH`? we don't appear to be using it in this script

Reply:
Done